lib/tasks/scheduler.rake

```diff
@@ -1,4 +1,4 @@
 desc 'Top up accounts automatically'
 task top_up_accounts: :environment do
-  AutomaticTopupService.new.perform if Date.current.friday?
+  AutomaticTopupsService.new.perform if Date.current.friday?
```

Comment:
What can we do to prevent this error? What has been done?

What to do with the failed job in production?